docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
+
+In all cases you can set CSS variables to override any value. By default the container `<div>` will have the attribute `id="sentry-feedback"`, meaning you can use the `#sentry-feedback` selector to define CSS variables to override defaults.
+
+Here is an example of customizing the background color for the light and dark themes by overriding CSS variables:
+
+```css
+#sentry-feedback {
+  --trigger-background: #cccccc;
+}
+@media (prefers-color-scheme: dark) {
+  #sentry-feedback {
+    --trigger-background: #222222;
+  }
+}
+```
+
+Or the same example above, by setting theme values in JavaScript:
+
+```javascript
+feedbackIntegration({
+  themeLight: {
+    background: "#cccccc",
+  },
+  themeDark: {
+    background: "#222222",
+  },
+});
+```
+
+The following values are only available as CSS variables, and apply to both dark and light themes.
+
+| Variable        | Default                                            | Description                                                                       |
+| --------------- | ---------------------------------------            | --------------------------------------------------------------------------------- |
+| `--font-family` | `"system-ui, 'Helvetica Neue', Arial, sans-serif"` | Default font-family to use                                                        |
+| `--font-size`   | `14px`                                             | Font size                                                                         |
+| `--z-index`     | `100000`                                           | The z-index of the widget                                                         |
+| `--inset`       | `auto 0 0 auto`                                    | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--page-margin` | `16px`                                             | How far from the edge of the screen should the widget be positioned.              |
 
-Colors can be customized via the Feedback class constructor or by defining CSS variables on the injected button. If you use the default button it will have a property `id="sentry-feedback`, meaning you can use the `#sentry-feedback` selector to define CSS variables to override.
+Colors can be customized by passing `themeLight` and/or `themeDark` to `feedbackIntegration({})`, or by defining CSS variables that override the default values.
 
 | key                       | css variable                   | light                                     | dark                                      | description                                                                                           |
 | ------------------------- | ------------------------------ | ----------------------------------------- | ----------------------------------------- | ----------------------------------------------------------------------------------------------------- |
```

Comment:
Can you make sure to add periods to the end of each description for consistency?